plugins/com.google.cloud.tools.eclipse.appengine.whitelist/src/com/google/cloud/tools/eclipse/appengine/whitelist/AppEngineJreWhitelist.java

```diff
@@ -1900,12 +1910,73 @@
               "javax.naming.ldap.Rdn"
               ));
   
+  // TODO: com.sun and com.oracle packages are tricky. Some are in the JRE. Some aren't. 
+  
   /**
-   * @param className fully package qualified className
-   * @return true if this class is allowed in Java 7 on App Engine, false otherwise
+   * @param className fully package qualified class name
+   * @return true if this class is allowed in Java 7 on App Engine Standard, false otherwise
    */
   public static boolean contains(String className) {
-    return WHITELIST.contains(className);
+    if (className.startsWith("javax.") ){
```

Comment:
Huh, good point: there are other javax.* implementations that can be carried along as a library.

I'm finding this logic a bit tortuous though.  Could we not merge the else-if logic (lines 1924-1931) into the isBundledInJre() and 
simplify it as:
```
return !isBundledInJre(className) || WHITELIST.contains(className));
```